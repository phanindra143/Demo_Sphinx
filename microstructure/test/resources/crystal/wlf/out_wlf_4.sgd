*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   144.6232
  !k_ij(1,2)=     0.2815
  !k_ij(2,2)=   144.6756
  !k_ij(1,3)=    -0.2747
  !k_ij(2,3)=     0.2438
  !k_ij(3,3)=   145.3208
MP,Kxx,1,  144.62320
MP,Kyy,1,  144.67562
MP,Kzz,1,  145.32080
MP,Kxx,2,  144.67562
MP,Kyy,2,  144.62320
MP,Kzz,2,  145.32080
MP,Kxx,3,  144.67562
MP,Kyy,3,  144.62320
MP,Kzz,3,  145.32080
MP,Kxx,4,  144.62320
MP,Kyy,4,  144.67562
MP,Kzz,4,  145.32080
MP,Kxx,5,  144.62320
MP,Kyy,5,  144.67562
MP,Kzz,5,  145.32080
MP,Kxx,6,  144.67562
MP,Kyy,6,  144.62320
MP,Kzz,6,  145.32080
MP,Kxx,7,  144.67562
MP,Kyy,7,  144.62320
MP,Kzz,7,  145.32080
MP,Kxx,8,  144.62320
MP,Kyy,8,  144.67562
MP,Kzz,8,  145.32080
MP,Kxx,9,  145.32080
MP,Kyy,9,  144.62320
MP,Kzz,9,  144.67562
MP,Kxx,10,  144.62320
MP,Kyy,10,  145.32080
MP,Kzz,10,  144.67562
MP,Kxx,11,  144.62320
MP,Kyy,11,  145.32080
MP,Kzz,11,  144.67562
MP,Kxx,12,  145.32080
MP,Kyy,12,  144.62320
MP,Kzz,12,  144.67562
MP,Kxx,13,  145.32080
MP,Kyy,13,  144.62320
MP,Kzz,13,  144.67562
MP,Kxx,14,  144.62320
MP,Kyy,14,  145.32080
MP,Kzz,14,  144.67562
MP,Kxx,15,  144.62320
MP,Kyy,15,  145.32080
MP,Kzz,15,  144.67562
MP,Kxx,16,  145.32080
MP,Kyy,16,  144.62320
MP,Kzz,16,  144.67562
MP,Kxx,17,  144.67562
MP,Kyy,17,  145.32080
MP,Kzz,17,  144.62320
MP,Kxx,18,  145.32080
MP,Kyy,18,  144.67562
MP,Kzz,18,  144.62320
MP,Kxx,19,  145.32080
MP,Kyy,19,  144.67562
MP,Kzz,19,  144.62320
MP,Kxx,20,  144.67562
MP,Kyy,20,  145.32080
MP,Kzz,20,  144.62320
MP,Kxx,21,  144.67562
MP,Kyy,21,  145.32080
MP,Kzz,21,  144.62320
MP,Kxx,22,  145.32080
MP,Kyy,22,  144.67562
MP,Kzz,22,  144.62320
MP,Kxx,23,  145.32080
MP,Kyy,23,  144.67562
MP,Kzz,23,  144.62320
MP,Kxx,24,  144.67562
MP,Kyy,24,  145.32080
MP,Kzz,24,  144.62320
*endif
