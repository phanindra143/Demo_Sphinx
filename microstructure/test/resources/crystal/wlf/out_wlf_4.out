            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\Rein_ohne_Poren\70-30
             3.00000      -2259708.55327         10632.72471          3555.55858
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000          -103.15864             3.66406
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000         14662.40098      -2260520.07117         -4873.09043
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000          -132.51894         15625.00000             6.00022
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000          2637.25977         -3902.05388      -2270634.28434
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000            13.01921             1.68052         15625.00000
  /%
